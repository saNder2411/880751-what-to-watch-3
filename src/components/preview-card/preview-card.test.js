import React from 'react';
import renderer from 'react-test-renderer';
import PreviewCard from './preview-card.jsx';

const previewCardTitle = `Bohemian Rhapsody`;

it(`Should PreviewCard render correctly`, () => {
  const markup = renderer
    .create(<PreviewCard previewCardTitle={previewCardTitle} />)
    .toJSON();

  expect(markup).toMatchSnapshot();
});
