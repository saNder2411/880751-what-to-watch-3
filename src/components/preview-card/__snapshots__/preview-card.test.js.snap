// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should PreviewCard render correctly 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  id={1}
  onClick={[Function]}
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <div
    className="small-movie-card__image"
  >
    <img
      alt="Bohemian Rhapsody"
      height="175"
      src="bohemian-rhapsody"
      width="280"
    />
  </div>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="movie-page.html"
    >
      Bohemian Rhapsody
    </a>
  </h3>
</article>
`;
